.circleci/config.yml

```diff
@@ -187,7 +187,8 @@ aliases:
 
   - &run-flow-checks
     name: Check for errors in code using Flow
-    command: yarn flow check
+    command: yarn flow-check-ios
+    command: yarn flow-check-android
```

Comment:
I think the second command overwrites the first. Looking at the CI logs for this step in this PR show that it only ran flow on android: https://circleci.com/gh/facebook/react-native/53561?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

You'll probably need to change this from `run-flow-checks` to `run-flow-ios` and have another one for android.

Reply:
Good catch, fixed that up